importer/ThTh.py

```diff
@@ -0,0 +1,83 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class ThTh(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the Admin Location.
+
+        First, try the District (amphoe). These are never linked,
+        so match against external list.
+        If failed, use the Province ISO code matched against external
+        list.
+        """
+        adm_q = None
+        prov_dic = self.data_files["provinces"]
+        distr_dic = self.data_files["districts"]
+        iso = self.prov_iso
+        district = self.district
+
+        dist_match = utils.get_item_from_dict_by_key(dict_name=distr_dic,
+                                                     search_term=district,
+                                                     search_in="itemLabel")
+        if len(dist_match) == 1:
+            adm_q = dist_match[0]
+
+        if adm_q is None:
+            prov_match = utils.get_item_from_dict_by_key(dict_name=prov_dic,
+                                                         search_term=iso,
+                                                         search_in="iso")
+            if len(prov_match) == 1:
+                adm_q = prov_match[0]
+
+        if adm_q is not None:
+            self.add_statement("located_adm", adm_q)
+        else:
+            self.add_to_report("district", self.district, "located_adm")
+
+    def set_heritage_id(self):
+        wlm = self.mapping["table_name"].upper()
+        self.add_statement("wlm_id", "{}-{}".format(wlm, self.register))
+
+    def update_labels(self):
+        th = utils.remove_markup(self.name)
+        self.add_label("th", th)
+
+    def update_descriptions(self):
+        en = "cultural heritage site in Thailand"
+        self.add_description("en", en)
+
+    def set_wd_from_name(self):
```

Comment:
I've corrected the harvesting so that these should now show up in `monument_article` after the next harvest.

Reply:
After the new harvest I believe this can be replaced by just `self.exists_with_monument_article(<lang code>)` since it's the default keyword.